tests/PoolTest.php

```diff
@@ -201,4 +202,36 @@ public function it_can_check_for_asynchronous_support()
     {
         $this->assertTrue(Pool::isSupported());
     }
+
+    /** @test */
+    public function it_can_run_synchronous_processes()
+    {
+        $pool = Pool::create();
+
+        foreach (range(1, 5) as $i) {
+            $pool->add(new SynchronousProcess(function () {
+                usleep(1000);
+
+                return 2;
+            }, $i))->then(function ($output) {
+                $this->assertEquals(2, $output);
+            });
+        }
+
+        $pool->wait();
```

Comment:
Should we assert here that the test took at least 5 seconds?
https://symfony.com/doc/current/components/stopwatch.html